api/rs/slint/Cargo.toml

```diff
@@ -167,6 +167,7 @@ log = { version = "0.4.17", optional = true }
 [dev-dependencies]
 slint-build = { path = "../build" }
 i-slint-backend-testing = { path = "../../../internal/backends/testing" }
+i-slint-backend-winit = { path = "../../../internal/backends/winit", features = ["wayland", "x11", "renderer-software"] }
```

Comment:
And this can't be tested with the testing backend?

Reply:
The testing backend doesn't provide an event loop, that's why I ended up with winit (which we also use for testing in the winit backend itself). If you'd like, I suppose I could try to implement a "dummy" event loop in the testing backend. Let me know :)

Reply:
I'd prefer that,  that would make the test much more lightweight

Reply:
Sounds good - shouldn't be hard (famous last words ;-)